### PR TITLE
Effects: Remove additional parameters of easings

### PR DESCRIPTION
**Pull request by Mr21**
Hi :)

A fix for #2367

### CODE REVIEW

File: src/effects/Tween.js [hunk 1]
```diff
@@ -27,17 +27,11 @@ Tween.prototype = {
...
-		if ( this.options.duration ) {
-			this.pos = eased = jQuery.easing[ this.easing ](
-				percent, this.options.duration * percent, 0, 1, this.options.duration
-			);
-		} else {
-			this.pos = eased = percent;
-		}
-		this.now = ( this.end - this.start ) * eased + this.start;
+		this.pos = this.options.duration ?
+			jQuery.easing[ this.easing ]( percent ) : percent;
```
Comment (gibson042) [inline src/effects/Tween.js]: The last `percent` should have its own line.

**Comment by gibson042** [comment]
I like it!

File: src/effects/Tween.js (hunk 1 above)
Comment (Mr21) [inline src/effects/Tween.js]: Done, thanks

**Comment by gnarf** [comment]
Could we try to land https://github.com/jquery/jquery/pull/2326 first and then you can update the test I wrote in there as well?

**Comment by Mr21** [comment]
After you ;)

**Comment by Mr21** [comment]
@gnarf, it's merged :)

File: test/unit/tween.js [hunk 2]
```diff
@@ -160,7 +160,7 @@ test( "jQuery.Tween - Plain Object", function() {
 
 	equal( tween.now, 90, "Calculated tween" );
 
-	ok( easingSpy.calledWith( 0.1, 0.1 * testOptions.duration, 0, 1, testOptions.duration ),
+	ok( easingSpy.calledWith( 0.1 ),
 		"...using jQuery.easing.linear with back-compat arguments" );
```
Comment (gnarf) [inline test/unit/tween.js]: can you update the comment here the "back-compat" arguments you removed :)

File: test/unit/tween.js [hunk 3]
```diff
@@ -196,7 +196,7 @@ test( "jQuery.Tween - Element", function() {
 	eased = 100 - ( jQuery.easing.swing( 0.1 ) * 100 );
 	equal( tween.now, eased, "Calculated tween" );
 
-	ok( easingSpy.calledWith( 0.1, 0.1 * testOptions.duration, 0, 1, testOptions.duration ),
+	ok( easingSpy.calledWith( 0.1 ),
 		"...using jQuery.easing.linear with back-compat arguments" );
```
Comment (gnarf) [inline test/unit/tween.js]: same

**Comment by gnarf** [comment]
Can we make an issue on the jQuery migrate plugin to add a shim to put the additional easing params back?

File: test/unit/tween.js (hunk 2 above)
Comment (Mr21) [inline test/unit/tween.js]: Done:
`ok( easingSpy.calledWith( 0.1 ), "...using jQuery.easing.linear" );`

**Comment by mzgol** [comment]
LGTM

**Comment by mzgol** [comment]
Thanks, landed.

**Pull request closed without merging**